hydra-main/src/main/java/com/addthis/hydra/job/spawn/Spawn.java

```diff
@@ -174,6 +184,8 @@
     private static final int LOG_MAX_AGE = Parameter.intValue("spawn.event.log.maxAge", 60 * 60 * 1000);
     private static final int LOG_MAX_SIZE = Parameter.intValue("spawn.event.log.maxSize", 100 * 1024 * 1024);
     private static final String LOG_DIR = Parameter.value("spawn.event.log.dir", "log");
+    private static final int EXPANDED_CONFIG_CACHE_MAX_SIZE_MB = Parameter.intValue("spawn.expandedConfig.maxSizeMb", 100);
```

Comment:
This way of specifying default parameters is deprecated. Remove these two lines and take a look at hydra-main/src/main/resources/spawn.conf you want to add the two parameters to that file.